### ISSUE TITLE
Is there a new release coming?

### ISSUE THREAD

**Issue by ugultopu** [mgbellemare/Arcade-Learning-Environment#201]
So far, there are two releases, in which latest is at October 2015. Is there a new release coming?

**Comment by mgbellemare** [comment]
Yes, I'd like to get 0.6 out the door before mid-June.

**Comment by nczempin** [comment]
@mgbellemare, could you create a milestone for 0.6 and assign some of the PRs and/or issues to it? At least for the ones I'm working on, if any are applicable, I can either try to get them done by then or at least let you know early if I'm not planning to work on them within that timeframe.

**Comment by tgpfeiffer** [comment]
I would love to see a new release, because the latest 0.5.1 does not install with Python 3 (99442f8 was merged after the release):

```
Processing /home/user/Arcade-Learning-Environment-0.5.1
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-zcqww0ua-build/setup.py", line 6
        print 'ERROR: Unable to find required library: %s. Please ensure you\'ve '\
                                                                                 ^
    SyntaxError: Missing parentheses in call to 'print'
```

**Comment by mgbellemare** [comment]
Hi Tobias,

There is a new version on the way, although it's been held up by a specific
PR. We could probably release the existing master as 0.5.2 -- is this what
you have in mind?

- Marc

**Comment by tgpfeiffer** [comment]
> We could probably release the existing master as 0.5.2
That would be great!

**Comment by tgpfeiffer** [comment]
Can I help anything to support you with making that 0.5.2 release?

**Comment by mgbellemare** [comment]
It should be fairly straightforward, but let me have a look. I'll aim to get back to you in a few days

**Comment by mgbellemare** [comment]
Well, ICML-17 happened and I didn't get a chance to look into this until now. I've released ALE 0.5.2 -- let me know if that works for you.

**Comment by tgpfeiffer** [comment]
Thank you very much, ALE 0.5.2 works great for me!

**Comment by mcmachado** [comment]
I'm closing this issue as we recently released the ALE 0.6.
